data/maps/SP_Scenarios/Along_The_River.wmf/scripting/objectives.lua

```diff
@@ -0,0 +1,93 @@
+-- =======================================================================
+--                             Along The River
+-- =======================================================================
+
+obj_defeat_red = {
+   name = "defeat_red",
+   title = _"Defeat Törver",
+   number = 1,
+   body = objective_text(_"Defeat the Armies of Törver.",
+      li_image("map:banner_plr_02.png", _[[Törver is the enemy in the north, who is fighting under the red banner.]])
+   ),
+}
+
+obj_build_port = {
+   name = "build_port",
+   title = _"Build a port",
+   number = 1,
+   body = objective_text(_"Build a Port in the North.",
+      li_image("tribes/buildings/warehouses/barbarians/port/menu.png", _[[We should build a port in Törvers former land to be ready to send an expedition as soon as we found out how to build ships.]])
+   ),
+}
+
+obj_conquer_pirate_sentry = {
+   name = "conquer_pirate_sentry",
+   title = _"Conquer pirate sentry",
+   number = 1,
+   body = objective_text(_"Conquer the Pirate Sentry.",
+      li_image("tribes/buildings/militarysites/barbarians/sentry/menu.png", _[[Conquer the pirate sentry near the brook to the east. Hopefully we will find out what happened to the gold in this area.]])
+   ),
+}
+
+obj_build_warehouse = {
+   name = "build_warehouse",
+   title = _"Build warehouse",
+   number = 1,
+   body = objective_text(_"Build a Warehouse near the Pirate Sentry.",
+      li_image("tribes/buildings/warehouses/barbarians/warehouse/menu.png", _[[Build a warehouse close to the conquered pirate sentry near the brook. We will need it as a basis for further exploitation. Furthermore we could use it to store some gold that we will mine from the surrounding hills.]])
```

Comment:
```suggestion
      li_image("tribes/buildings/warehouses/barbarians/warehouse/menu.png", _[[Build a warehouse close to the conquered pirate sentry near the brook. We will need it as a base for further exploitation. Furthermore we could use it to store some gold that we will mine from the surrounding hills.]])
```
exploitation or exploration?